specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -79,6 +79,48 @@
         }
       }
     },
+    "/keys/{key-name}/rotate": {
+      "put": {
```

Comment:
Should be `POST`. The `PUT` method is designed for URIs that represent a resource and supports `GET`.

The `/rotate` is an action, not a resource (i.e. there will never be `GET /keys/{key-name}/rotate`).